Lib/timeit.py

```diff
@@ -243,6 +244,64 @@ def repeat(stmt="pass", setup="pass", timer=default_timer,
     return Timer(stmt, setup, timer, globals).repeat(repeat, number)
 
 
+def _run(t, repeat=default_repeat, number=None, time_unit=None, verbose=False, precision=3):
+    if number == None:
```

Comment:
Identity check is faster (maybe check with `timeit.run()` 😄):

```suggestion
    if number is None:
```